Sources/Core/JavaModelRenderer.swift

```diff
@@ -351,21 +361,7 @@ public struct JavaModelRenderer: JavaFileRenderer {
                                 "builder.set" + Languages.java.snakeCaseToCapitalizedPropertyName(param) + "(this." + typeAdapterVariableNameForType(unwrappedTypeFromSchema(param, schemaObj.schema)) + ".read(reader));",
                             ]
                         )
-                    } + [
```

Comment:
At line 335 above there's `boolean[] bits = null;` which can be removed.
And at line 372 below there's the corresponding use of it that can also be removed:

```
JavaIR.ifBlock(condition: "bits != null") { ...
```

(GitHub doesn't seem to let me add comments outside of the changed lines)